parquet/src/test/java/org/apache/iceberg/parquet/ParquetConversionsTest.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iceberg.parquet;
+
+import org.apache.iceberg.expressions.Literal;
+import org.apache.iceberg.types.Types;
+import org.apache.parquet.io.api.Binary;
+import org.apache.parquet.schema.LogicalTypeAnnotation;
+import org.apache.parquet.schema.PrimitiveType;
+import org.apache.parquet.schema.Type;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class ParquetConversionsTest {
+  @Test
+  public void fromParquetPrimitive() {
+    Literal<Object> out = ParquetConversions.fromParquetPrimitive(
+        Types.StringType.get(),
+        org.apache.parquet.schema.Types.primitive(PrimitiveType.PrimitiveTypeName.BINARY, Type.Repetition.OPTIONAL)
+            .as(LogicalTypeAnnotation.enumType())
+            .named("enum"),
+        Binary.fromString("value")
+    );
+    Assert.assertEquals(Literal.of("value"), out);
+  }
+}
```

Comment:
It seems a bit odd to just have tests for the ParquetConversions, I took a look in the code and if I'm not mistaken `ParquetConversions` utilities are ultimately used for footer metrics? 

If that's the case, then I think it makes more sense to test `ParquetUtil.fileMetrics`. I don't see any existing tests, although maybe we indirectly assert the metrics during data file write tests. So we may just want to take this opportunity to start the test for `ParquetUtil.fileMetrics` or if there's existing tests which assert the metrics then we can leverage those ones already. What are your thoughts on this?

Reply:
Hmm yeah it's definitely possible to test it at a higher level. The only problem is that enum isn't an Iceberg type so I think the test would have to create the Parquet file directly with Parquet API's first, whereas other tests usually deal with Iceberg APIs.